src/main/java/com/example/calculator/CalculatorController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.calculator;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CalculatorController {
+    private final CalculatorService calculatorService;
+
+    public CalculatorController(CalculatorServiceImpl calculatorServiceImpl) {
```

Comment:
```suggestion
    public CalculatorController(CalculatorService calculatorService) {
```
надо использовать всегда интерфейсные ссылки